### ISSUE TITLE
Should samplesheet support absolute path to data?

### ISSUE THREAD

**Issue by necrolyte2** [VDBWRAIR/ngs_mapper#211]
Watching new users use the pipeline and training them made me realize that the samplesheet seems a bit odd in the way it interacts with runsamplesheet.sh.

Right now you provide a path to where you have directories that have read data(fastq and/or sff)
`runsamplesheet.sh /path/to/data /path/to/samplesheet.tsv`

Should we just make runsamplesheet.sh have 1 parameter(samplesheet.tsv) and then move the path into that first column?

**Comment by necrolyte2** [comment]
If we do decide to support this, then we would need to have runsamplesheet.sh support both modes.
That is, detect if `$1` is a samplesheet.tsv path or a directory that contains samples and run accordingly

**Comment by averagehat** [comment]
Is it possible that the samplesheet and the reads are in different directories?

**Comment by necrolyte2** [comment]
Yes. Currently the runsamplesheet.sh supports relative and absolute path for $1
